templates/scheduler/scheduler-deployment.yaml

```diff
@@ -8,7 +8,7 @@
 {{- $kube := eq .Values.executor "KubernetesExecutor" }}
 # Is persistence enabled on the _workers_?
 # This is important because in $local mode, the scheduler assumes the role of the worker
-{{- $persistence := .Values.workers.persistence.enabled }}
```

Comment:
@schnie wasn't sure what to do here. Do we make persistence global across all queues?

Reply:
Hmm, its really a "legacy" feature of the chart. This was back when we deployed the workers as a StatefulSet and they stored the logs on that volume. These days we're using centralized logging (Elastic now, Buckets in next gen), so this feature isn't as important from a platform perspective. Not sure if anyone in OSS benefits from this or not.

We set it to false, but since it has to do with logging, maybe it belongs as a "global" queue setting.